src/system.js

```diff
@@ -136,5 +136,6 @@ export {
   Toggle,
   TooltipNotice,
   UrlBar,
-  VirtualItem
+  VirtualItem,
+  Guard
```

Comment:
would you mind keeping in alphabetical order? just easier to scan list that way

Reply:
Done, sorted alphabetically.